ArduCopter/mode_guided.cpp

```diff
@@ -844,6 +854,36 @@ void ModeGuided::velaccel_control_run()
     }
 }
 
+// velaccel_control_run - runs the guided velocity and acceleration controller
```

Comment:
small copy-paste issue with the comment.

Reply:
Done.